### DISCUSSION TITLE
Delete files across backups

### DISCUSSION THREAD

**Discussion by plrnr** [nextcloud/all-in-one#610] [Questions] [answered]
Hi,

It seems like the AIO backup does incremental backups giving the ability to restore to a point in time. This would make backups keep its size. 
Here is my use case - I purge unwanted files regularly such as unwanted photos. I would also like these to be purged out of backups.
How do I achieve that?

1. Basically, I plan to save the backup to an external drive to avoid single point of failure. So my goal here is not so much about being able to restore to a point in time but just to avoid SPF.

2. How do I backup raw data without encryption/compression? I would like to mirror my data to external drive.

**Comment by szaimen** [comment, marked as answer]
Hi, first of all see this discussion: https://github.com/nextcloud/all-in-one/discussions/428#discussioncomment-2496070
And second see this: https://github.com/nextcloud/all-in-one#delete-backup-archives-manually

**Reply by plrnr** [reply]
@szaimen 
Many thanks for answering. 

In addition, I read a bit on borg so it seems like the archives are not dependent on each other. 
So if I have 3 archives and I delete the first one, the other two have backups fully accessible.. correct?

The AIO backup provides daily backup... is there a way to call it on demand? use case: I could upload 100GB data at one point, ideally I would like to perform a backup right away.. so that this data is copied over to my external drive. If not, I run a risk having to wait till the AIO scheduled backup kicks in. What are my options for this?

**Reply by szaimen** [reply]
> So if I have 3 archives and I delete the first one, the other two have backups fully accessible.. correct?

Correct



> The AIO backup provides daily backup... is there a way to call it on demand? use case: I could upload 100GB data at one point, ideally I would like to perform a backup right away.. so that this data is copied over to my external drive. If not, I run a risk having to wait till the AIO scheduled backup kicks in. What are my options for this?

Just simply create a backup manually from the aio interface then?

**Reply by plrnr** [reply]
Yes, I missed that obviously! Many thanks Simon!

**Comment by szaimen** [comment]
> 2\. How do I backup raw data without encryption/compression? I would like to mirror my data to external drive.

This is not possibke and will not be implemented. However you may be able to use the Nextcloud Backup app for this. See https://apps.nextcloud.com/apps/backup

**Reply by szaimen** [reply]
Apart from this option you may just use rsync or other tools to sync all files and data unencrypted and uncompressed to a different location.

**Reply by plrnr** [reply]
Could you please point me to the actual directory path where the  data is stored on the host?

**Reply by plrnr** [reply]
@szaimen can you please answer this.. thank you

**Reply by szaimen** [reply]
> Could you please point me to the actual directory path where the data is stored on the host?

On Linux it should be subdirectories of `/var/lib/docker/volumes`...
(When not adjusting NEXTCLOUD_DATADIR)
